backend/tests/conftest.py

```diff
@@ -120,7 +122,8 @@ async def reset_registry(db: InfrahubDatabase) -> None:
 
 
 @pytest.fixture
-async def default_branch(reset_registry, local_storage_dir, empty_database, db: InfrahubDatabase) -> Branch:
```

Comment:
I'm surprised by this change, this fixture is used by hundreds of tests, are you sure of the impact of this change ? and more importantly please can you provide more information why this change is required

Reply:
I actually did not mean to push these changes, I did that for local testing purposes, nice catch thanks